content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,306 @@
+disrupted by consolidation. Consolidation should never remove a Node where a Pod annotated
+with `autoscaling.kubernetes.io/do-not-disrupt: true` is running.
+
+Example:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example
+  annotations:
+    autoscaling.kubernetes.io/do-not-disrupt: true
+  labels:
+    app: example
+spec:
+  containers:
+    - name: pause
+      image: pause
+```
+
+## Combine workload and Node autoscaling
+
+### Horizontal workload autoscaling {#horizontal-workload-autoscaling}
+
+Node autoscaling usually works in response to Pods - it provisions new Nodes to accommodate
+unschedulable Pods, and then consolidates the Nodes once they're no longer needed.
+
+[Horizontal workload autoscaling](/docs/concepts/workloads/autoscaling#scaling-workloads-horizontally)
+automatically scales the number of workload replicas to maintain a desired average resource
+utilization across the replicas. In other words, it automatically creates newPods in response to
+application load, and then removes the Pods once the load decreases.
+
+You can use Node autoscaling together with horizontal workload autoscaling to autoscale the Nodes in
+your cluster based on the average real resource utilization of your Pods.
+
+If the application load increases, the average utilization of its Pods should also increase,
+prompting workload autoscaling to create new Pods. Node autoscaling should then provision new Nodes
+to accommodate the new Pods.
+
+Once the application load decreases, workload autoscaling should remove unnecessary Pods. Node
+autoscaling should in turn consolidate the Nodes that are no longer needed.
+
+If configured correctly, this pattern ensures that your application always has the Node capacity to
+handle load spikes if needed, but you don't have to pay for the capacity when it's not needed.
+
+### Vertical workload autoscaling {#vertical-workload-autoscaling}
+
+When using Node autoscaling, it's important to set Pod resource requests correctly. If the requests
+of a given Pod are too low, provisioning a new Node for it might not help the Pod actually run.
+If the requests of a given Pod are too high, it might incorrectly prevent consolidating its Node.
+
+[Vertical workload autoscaling](/docs/concepts/workloads/autoscaling#scaling-workloads-vertically)
+automatically adjusts the resource requests of your Pods based on their historical resource usage.
+
+You can use Node autoscaling together with vertical workload autoscaling in order to adjust the
+resource requests of your Pods while preserving Node autoscaling capabilities in your cluster.
+
+{{< warning >}}
+When using Node autoscaling, it's not recommended to set up vertical workload autoscaling for
+DaemonSet Pods. Autoscalers have to predict how DaemonSet Pods on a new Node will look like in order
+to predict available Node resources. Vertical workload autoscaling might make these predictions
+unreliable, leading to incorrect scaling decisions.
+{{</ warning >}}
```

Comment:
We'd use `caution` here I think (we == SIG Docs). `warning` is reserved for “pay attention, or your overall information security could be at risk”.

Reply:
Makes sense, thanks